controllers/admin/AdminPerformanceController.php

```diff
@@ -614,7 +614,15 @@ public function postProcess()
             return;
         }
 
+        /**
+         * /!\ Dont forget to migrate the HOOK
+         */
+        dump('action'.get_class($this).ucfirst($this->action).'Before');die;
```

Comment:
Note this is a **BC**, we can't return the same instance of `controller` than before as I will remove the old `AdminPerformancesController` and the new one is really different: for integrators, why do they need an instance of the controller?